kivy/core/window/window_sdl2.py

```diff
@@ -864,9 +863,8 @@ def ungrab_mouse(self):
     def set_custom_titlebar(self, titlebar_widget):
         if self.borderless and platform == 'win':
```

Comment:
I think this needs better handling, as borderless breaks the API only on windows, and say if someone is developing on Linux they can use both `Windows.borderless` and `Window.custom_titlebar`  and when the same code is executed on windows it will break the program